README.md

```diff
@@ -80,6 +80,7 @@ correct Kubernetes API group and version to use based on manifests:
 const K8Api = require('kubernetes-client');
 const api = new K8Api.Api({
   url: 'http://my-k8-api-server.com',
+  namespace: 'test-ns'
```

Comment:
we use `namespace: 'my-project'` in the example above.  Can you use `'my-project'` here too?  or change other uses to `'test-ns'`?